app/views/static/references.html.erb

```diff
@@ -9,30 +9,81 @@
 
-        <p class='static-page__text static-page__text--sinai'>Kamil = Kāmil, Murād. <i>Catalogue of All Manuscripts in the Monastery of St. Catherine on Mount Sinai.</i> Harrassowitz, 1970. [<%= link_to 'WorldCat', "https://www.worldcat.org/title/catalogue-of-all-manuscripts-in-the-monastery-of-st-catharine-on-mount-sinai/oclc/859613867?referer=br&ht=edition", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>G&eacute;hin 2006= G&eacute;hin, Paul. &ldquo;Manuscrits sina&iuml;tiques dispers&eacute;s, I: les fragments arabes et syriaques de Paris.&rdquo; <i>OC</i>90 (2006): 23-43. [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/-manuscrits-sinaitiques-disperses-i-les-fragments-syriaques-et-arabes-de-paris/oclc/784157423%26referer%3Dbrief_results&amp;sa=D&amp;source=editors&amp;ust=1683922505979436&amp;usg=AOvVaw1uYIf9GxuZi6FqmEf8cZfR"', class: 'si-link' %>]</p>
 
-        <p class='static-page__text static-page__text--sinai'>Lewis = Lewis, Agnes Smith. <i>Catalogue of the Syriac Manuscripts in the Convent of Saint Catherine on Mount Sinai.</i> Cambridge University Press, 1894. [<%= link_to 'Internet Archive', "https://archive.org/details/catalogueofsyria00sainrich", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>G&eacute;hin 2009 = G&eacute;hin, Paul. &quot;Fragments patristiques syriaques des Nouvelles d&eacute;couvertes du Sina&iuml;.&quot; <i>Collectanea Christiana Orientalia</i>6 (2009): 67-93</p>
 
-        <p class='static-page__text static-page__text--sinai'>Meimaris = Meimaris, Yiannis E. <i>Katalogos ton neon aravikon cheirographon tes Hieras Mones Hagias Aikaterines tou Orous Sina = Kataluy al-Majtutat al-’arabiyya al-muktasafa haditan bi-Dayr Sant Katarin al-muqaddas.</i> Ethnikon Hidryma Ereunon, 1985. [<%= link_to 'WorldCat', "https://www.worldcat.org/title/katalogos-ton-neon-aravikon-cheirographon-tes-hieras-mones-hagias-aikaterines-tou-orous-sina-kataluy-al-majtutat-al-arabiyya-al-muktasafa-haditan-bi-dayr-sant-katarin-al-muqaddas/oclc/1123922473?referer=br&ht=edition", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>G&eacute;hin 2017 = G&eacute;hin, Paul. <i>Les manuscrits syriaques de parchemin du Sina&iuml; et leurs</i>membra disjecta (CSCO 665, Subsidia 136; Leuven, 2017). [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/manuscrits-syriaques-de-parchemin-du-sinai-et-leurs-membra-disjecta/oclc/1000455293%26referer%3Dbrief_results&amp;sa=D&amp;source=editors&amp;ust=1683922505980598&amp;usg=AOvVaw2ZbqofVswP2V57Y1V5Z1Z_"', class: 'si-link' %>]</p>
```

Comment:
Missing space: `et leurs</i>membra` -> `et leurs</i> membra`

Reply:
URL should be: https://www.worldcat.org/title/manuscrits-syriaques-de-parchemin-du-sinai-et-leurs-membra-disjecta/oclc/1000455293